styles/styles.css

```diff
@@ -802,6 +802,12 @@ main .section.static .section-container h1 {
   font-weight: var(--font-weight-medium);
 }
 
+/* section - bottom-light-border */
```

Comment:
This seems to be an `h2` style rather than a `section` level style. You can try putting this `h2` in a `text` block with a bottom border variant (i think it has one).